daml-assistant/integration-tests/BUILD.bazel

```diff
@@ -132,3 +140,63 @@ da_haskell_test(
         "//libs-haskell/test-utils",
     ],
 )
+
+da_haskell_test(
+    name = "create-daml-app-tests",
+    timeout = "long",
+    srcs = [
+        "src/DA/Daml/Assistant/CreateDamlAppTests.hs",
+        "src/DA/Daml/Assistant/IntegrationTestUtils.hs",
```

Comment:
Can we please factor this out into a library instead of trying to compile it twice.

Reply:
Done, thanks